acceptance-tests/src/test/kotlin/com/chutneytesting/acceptance/common/ReportSteps.kt

```diff
@@ -0,0 +1,30 @@
+/*
+ * SPDX-FileCopyrightText: 2017-2024 Enedis
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ */
+
+package com.chutneytesting.acceptance.common
+
+import com.chutneytesting.kotlin.dsl.ChutneyStepBuilder
+import com.chutneytesting.kotlin.dsl.CompareAction
+import com.chutneytesting.kotlin.dsl.spEL
+
+fun ChutneyStepBuilder.checkScenarioSuccess() {
+  CompareAction(
+      mode = "equals",
+      actual = "json(#report, \"$.report.status\")".spEL,
```

Comment:
For spEL in java String, to avoid to escape `"`, you could use `'`, as `json(#report, '$.report.status')`